OMCompiler/Compiler/Util/Autoconf.mo.omdev.mingw

```diff
@@ -9,11 +10,20 @@ encapsulated package Autoconf
   constant String make = "make";
   constant String exeExt = ".exe";
   constant String dllExt = ".dll";
-  constant String ldflags_basic = " -Wl,-Bstatic -lomcgc -lregex -ltre -lintl -liconv -lexpat -static-libgcc -luuid -loleaut32 -lole32 -limagehlp -lws2_32 -llis -lumfpack -lklu -lcolamd -lbtf -lamd  -lsundials_idas -lsundials_cvodes -lsundials_kinsol -lsundials_nvecserial -lipopt -lcoinmumps -lpthread -lm -lgfortranbegin -lgfortran -lquadmath -lmingw32 -lgcc_eh -lmoldname -lmingwex -lmsvcrt -luser32 -lkernel32 -ladvapi32 -lshell32 -lopenblas -lcminpack -Wl,-Bdynamic";
+  constant String libFortran = if not isNewOMDev then "-lgfortranbegin " else " ";
+  constant String libMsvcrt = if not isNewOMDev then "-lmsvcrt " else " ";
+  constant String ldflags_basic = " -Wl,-Bstatic -lomcgc -lregex -ltre -lintl -liconv -lexpat -static-libgcc -luuid -loleaut32 -lole32 -limagehlp -lws2_32 -llis -lumfpack -lklu -lcolamd -lbtf -lamd  -lsundials_idas -lsundials_cvodes -lsundials_kinsol -lsundials_nvecserial -lipopt -lcoinmumps -lpthread -lm " +
```

Comment:
On MINGW we will generate a static and dynamic Sundials (at least I think so) , on MSVC we only generate the static on (which should not really be working though, because KLU and LAPCK is not linked correctly).
That could be a problem with adding `-Wl,-Bstatic`.

Reply:
I think there are some other flags for MSVC. hardcoded in tpl that generates the makefile.